### DISCUSSION TITLE
[Feature Request] Just Like The Medical NPC, a periscope Gun NPC that refills submarine ammo for periscopes.

### DISCUSSION THREAD

**Discussion by GreatBlueMoon** [FakeFishGames/Barotrauma#9788] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

The option to use a menu at an outpost particularly military outposts to refill ammo boxes for submarine periscope guns, similarly to that of the medical menu you can heal all the crewmates at by spending bits of marks, probably a little more pricier though.

### Reproduction steps

1. Go to outpost
2. Approach NPC with coilgun ammo icon
3. Pay to refill the ammo in select rooms of the submarine, clicking on a submarine room and then it'l show a list of the coilgun or whatever boxes currently inside and a bar to represent how full they are.

### Bug prevalence

Happens every time I play

### Version

Other

### -

Suggestion

### Which operating system did you encounter this bug on?

Windows

**Comment by ahzidaljun** [comment]
a cool idea, and it would help limit spare or empty ammo cluttering. recycle recipes are a thing and they have a pretty minimal cost, but so is using medicine and the medic npcs were added for a reason.

**Comment by GreatBlueMoon** [comment]
been playing the game for years and i dont know if this bumps this suggestion, but ammo boxes have always been a clutter problem, especially in singleplayer but, even in multiplayer its less about the gameplay and more the annoyance of managing it all, a feature as such would help with the problem a bit, especially if we could even sell the empty boxes separate from full ones via the npc

**Comment by GreatBlueMoon** [comment]
another bump

**Comment by bookslayer10** [comment]
Just paying marks to refill the boxes seems a bit OP. In general, aquiring more ammo requires multiple stations worth of resources due to scarcity, so just being able to infinitely refill ammo boxes through just marks would be too powerful. For example, you can't buy physicorium at most stations, but this suggestion would allow you to refill as many physicorium ammo boxes as you can afford.

**Reply by GreatBlueMoon** [reply]
> Just paying marks to refill the boxes seems a bit OP. In general, aquiring more ammo requires multiple stations worth of resources due to scarcity, so just being able to infinitely refill ammo boxes through just marks would be too powerful. For example, you can't buy physicorium at most stations, but this suggestion would allow you to refill as many physicorium ammo boxes as you can afford.

i know this post is old and this is a old reply, but after many many runs involving ships flawed by intentional design like the dudong not involving fabricators, a ammo refilling npc would still be a great idea i think, only bumping this thread for the suggestive idea again.

As for phychorium and other ammo types, perhaps make it the same price as filling up a coilgun ammo, which sounds insane, but the kicker is that it automatically uses the phychorium stored in your ship, that way you can get by without a fabricator and do it quickly and still pay for the resource and not just in marks.
